contents/handbook/people/team-structure/core-experience.md

```diff
@@ -24,51 +27,48 @@ organization in matters of product design, user experience, and frontend & API t
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
+
 ## Mission
 
+Company goal that resonates most with this team:
+
+> Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch.
+
 Team Core Experience aims to:
 
-- Implement PostHog's vision of increasing the number of successful products in the world, by building features that will move us closer to said vision.
+- Increase the quality of our features across all dimensions (see scope below).
```

Comment:
This came up as a distinction between core experience and core analytics - core experience's contributions to user experience are often at a system level and span across features. Said another way, core experience is responsible for making sure all features work well together as a cohesive product. Thoughts?

Reply:
> "core experience's contributions to user experience are often at a system level and span across features. Said another way, core experience is responsible for making sure all features work well together as a cohesive product. "

That's a nice framing.

Feels like you could use it to test if you're getting off track too. As a hypothetical example: "We've spent a lot of time ${working on the specifics of how paths works} recently. We should focus on ${helping people get to and from paths} instead"

Reply:
Definitely think this helps clarify things, 👍 to the general idea!